src/helpers.rs

```diff
@@ -1,8 +1,11 @@
 use std::mem;
 
-use rustc::ty::{self, layout::{self, Size, Align}};
 use rustc::hir::def_id::{DefId, CRATE_DEF_INDEX};
 use rustc::mir;
+use rustc::ty::{
+    self,
+    layout::{self, Align, LayoutOf, Size},
+};
```

Comment:
Looks like you ran rustfmt on this?  Please make that a separate PR, all these spurious changes make reviewing the real changes much harder.

Reply:
Yes, i isolated it into a single commit so it was easier to review. I'll try to remove the commit without blowing this into smithereens